cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -31,61 +31,66 @@ $$ LANGUAGE plpgsql;
 -- Census-block-level data. TODO: consider renaming the table.
 
 CREATE TABLE IF NOT EXISTS demographics(
-    census_geo_id varchar(255) NOT NULL,
-    census_block_name varchar(255),
-    total_population real,
+    census_geo_id         varchar(255) NOT NULL,
+    census_block_name     varchar(255),
+    total_population      real,
     under_five_population real,
-    poverty_population real,
-    black_population real,
-    white_population real,
-    geom GEOMETRY(Geometry, 4326),
-    PRIMARY KEY(census_geo_id)
+    poverty_population    real,
+    black_population      real,
+    white_population      real,
+    geom                  GEOMETRY(Geometry, 4326),
+    PRIMARY KEY (census_geo_id)
 );
 
 CREATE INDEX IF NOT EXISTS geom_index
     ON demographics
-    USING GIST (geom);
+        USING GIST (geom);
 
 -- Water-system-level data
 
-CREATE TABLE IF NOT EXISTS water_systems(
-    pws_id varchar(255) NOT NULL,
-    lead_connections_count real,
-    geom GEOMETRY(Geometry, 4326),
-    PRIMARY KEY(pws_id)
-    );
-
-ALTER TABLE water_systems
-    ADD COLUMN IF NOT EXISTS pws_name varchar(255),
-    ADD COLUMN IF NOT EXISTS service_connections_count real,
-    ADD COLUMN IF NOT EXISTS population_served real;
+CREATE TABLE IF NOT EXISTS water_systems
+(
+    pws_id                    varchar(255) NOT NULL,
+    lead_connections_count    real,
+    geom                      GEOMETRY(Geometry, 4326),
```

Comment:
Optional nit: can you move geom to the end?

Reply:
Done.